Cargo.toml

```diff
@@ -25,19 +26,23 @@ resolver = "2"
 [dependencies]
 rspotify-macros = { path = "rspotify-macros", version = "0.10.0" }
 rspotify-model = { path = "rspotify-model", version = "0.10.0" }
+rspotify-http = { path = "rspotify-http", version = "0.10.0", default-features = false }
```

Comment:
Because we need to manually specify which client to use. Perhaps `rspotify-http` shouldn't enable clients by default, and that should be done by `rspotify` itself?